Code/Tools/TestImpactFramework/Runtime/Common/Code/Include/Static/Process/JobRunner/TestImpactProcessJobRunner.h

```diff
@@ -45,21 +46,45 @@ namespace TestImpact
         //! @param jobDataMap The job data (in the form of job info and meta-data) for each job run.
         using PayloadMapProducer = AZStd::function<PayloadMap(const JobDataMap& jobDataMap)>;
 
-        //! Callback for job completion/failure.
-        //! @param jobInfo The job information associated with this job.
-        //! @param meta The meta-data about the job run.
-        //! @param std The standard output and standard error of the process running the job.
-        using JobCallback =
-            AZStd::function<ProcessCallbackResult(const typename Job::Info& jobInfo, const JobMeta& meta, StdContent&& std)>;
-
-        //! Callback for job standard output/error buffer consumption in real-time.
-        //! @note The full standard output/error data is available to all capturing jobs at their end of life regardless of this callback.
-        //! @param jobInfo The job information associated with this job.
-        //! @param stdOutput The total accumulated standard output buffer.
-        //! @param stdError The total accumulated standard error buffer.
-        //! @param stdOutDelta The standard output/error buffer data since the last callback.
-        using StdContentCallback = AZStd::function<ProcessCallbackResult(
-            const typename Job::Info& jobInfo, const AZStd::string& stdOutput, const AZStd::string& stdError, AZStd::string&& stdOutDelta, AZStd::string&& stdErrDelta)>;
+        //! Bus for job runner notifications.
+        class Notifications
```

Comment:
suggestion: It might be better to have a more specific name than this

Reply:
I did toy with that initially but figured as consumers access it through the class it was as descriptive but more succinct, i.e. (i.e. `NativeTestRunner::NotificationsBus` vs `NativeTestRunner::JobRunnerNotificationsBus`).

Reply:
Btw fixed the other comments... not sure why they don't have their own reply box!